react_practice/react-apps/waitlist-page-v1/src/PrimaryBtn.css

```diff
@@ -1,21 +1,18 @@
-.primary-btn-title {
+.primary-btn {
```

Comment:
I named this class `primary-btn`. You'll noticed I changed the DOM nodes used in the `PrimaryBtn` class to remove the `div` and `h5` (see my comments below on why). This being the case, I found it a bit nicer to name the class after the component to better match.